packages/android_intent_plus/example/lib/main.dart

```diff
@@ -36,6 +36,8 @@ class MyApp extends StatelessWidget {
 /// Holds the different intent widgets.
 class MyHomePage extends StatelessWidget {
   const MyHomePage({super.key});
+  static const _intent =
+      'intent://payment#Intent;action=ch.twint.action.TWINT_PAYMENT;scheme=twint;S.code=98815;S.startingOrigin=EXTERNAL_WEB_BROWSER;S.browser_fallback_url=;end';
```

Comment:
Let's replace this with a generic intent URI something like `example://example`.

Reply:
That's not a valid intent URI, intent URI should start with intent, I have used toUri to generate the same alarm intent as the first button so both should do the same now.